backend/benefit/calculator/tests/test_previous_benefits_api.py

```diff
@@ -88,6 +94,7 @@ def test_create_previous_benefit_unauthenticated(anonymous_client, previous_bene
     assert response.status_code == 403
 
 
+@override_settings(NEXT_PUBLIC_MOCK_FLAG=False)
```

Comment:
There is so many of these settings overrides that could it be easier to set it false to the default fixture settings?

Reply:
Looked into this and it looks like [bf-pytest.yml](https://github.com/City-of-Helsinki/yjdh/blob/b404c6dc13763037b01a369c454b65814ecb73ca/.github/workflows/bf-pytest.yml) didn't set DISABLE_AUTHENTICATION or NEXT_PUBLIC_MOCK_FLAG to any value. Based on that I'd think it would have used [NEXT_PUBLIC_MOCK_FLAG=False](https://github.com/City-of-Helsinki/yjdh/blob/b404c6dc13763037b01a369c454b65814ecb73ca/backend/benefit/helsinkibenefit/settings.py#L53) and [DISABLE_AUTHENTICATION=False](https://github.com/City-of-Helsinki/yjdh/blob/b404c6dc13763037b01a369c454b65814ecb73ca/backend/benefit/helsinkibenefit/settings.py#L108) which were their default values in Helsinki benefit's [settings.py](https://github.com/City-of-Helsinki/yjdh/blob/b404c6dc13763037b01a369c454b65814ecb73ca/backend/benefit/helsinkibenefit/settings.py). So the backend tests were run with such values in the review environment.

Reply:
Created #1483 related to this.